consensus/misc/eip2935.go

```diff
@@ -29,6 +30,10 @@ import (
 )
 
 func StoreBlockHashesEip2935(header *types.Header, state *state.IntraBlockState, config *chain.Config, headerReader consensus.ChainHeaderReader) {
+	if state.GetCodeSize(params.HistoryStorageAddress) == 0 {
```

Comment:
`GetCodeHash` is slightly more efficient than `GetCodeSize`

Reply:
Do you have a benchmark for that? I ran a short benchmark to test the above. I can conclusively say that if the state object is hot (for most of its life after the fork), `GetCodeSize` is actually more efficient than `GetCodeHash` for `intra_block_state` up to quite a long code size (400KB-ish) 
It is also significantly more efficient if the additional `if` conditions were to be factored in, for this check